src/test/java/org/dependencytrack/tasks/CloneProjectTaskTest.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * This file is part of Dependency-Track.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * Copyright (c) OWASP Foundation. All Rights Reserved.
+ */
+package org.dependencytrack.tasks;
+
+import org.dependencytrack.PersistenceCapableTest;
+import org.dependencytrack.event.CloneProjectEvent;
+import org.dependencytrack.model.Project;
+import org.dependencytrack.resources.v1.vo.CloneProjectRequest;
+import org.junit.Test;
+
+import java.time.Instant;
+import java.util.Date;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.dependencytrack.model.WorkflowStatus.COMPLETED;
+import static org.dependencytrack.model.WorkflowStep.PROJECT_CLONE;
+
+public class CloneProjectTaskTest extends PersistenceCapableTest {
+
+    @Test
+    public void testCloneProjectTask() {
```

Comment:
I think this should be covered by the clone tests in `ProjectResourceTest` already?

Reply:
Yes but Codacy test coverage was complaining about the `CloneProjectTask` although the code is covered by the `ProjectResourceTest`.

Reply:
Do you see the same missing coverage when using [Run with Coverage](https://www.jetbrains.com/help/idea/code-coverage.html#run_with_coverage) on the resource test?

Reply:
yeah locally `CloneProjectTask` is almost fully covered by `ProjectResourceTest`.